src/test/kotlin/com/terraformation/backend/customer/db/UserStoreTest.kt

```diff
@@ -753,18 +753,44 @@ internal class UserStoreTest : DatabaseTest(), RunsAsUser {
       val userId = insertUser(10)
       insertUserGlobalRole(userId, GlobalRole.AcceleratorAdmin)
 
-      userStore.updateGlobalRoles(userId, setOf(GlobalRole.SuperAdmin))
+      userStore.updateGlobalRoles(setOf(userId), setOf(GlobalRole.SuperAdmin))
 
       assertEquals(
           listOf(UserGlobalRolesRow(userId = userId, globalRoleId = GlobalRole.SuperAdmin)),
           userGlobalRolesDao.findAll())
     }
 
+    @Test
+    fun `overwrites existing global roles for a set of users`() {
+      val userId1 = insertUser(10)
+      val userId2 = insertUser(11)
+      insertUserGlobalRole(userId1, GlobalRole.AcceleratorAdmin)
+      insertUserGlobalRole(userId2, GlobalRole.TFExpert)
+
+      userStore.updateGlobalRoles(setOf(userId1, userId2), setOf(GlobalRole.SuperAdmin))
+
+      assertEquals(
+          listOf(
+              UserGlobalRolesRow(userId = userId1, globalRoleId = GlobalRole.SuperAdmin),
+              UserGlobalRolesRow(userId = userId2, globalRoleId = GlobalRole.SuperAdmin)),
+          userGlobalRolesDao.findAll())
+    }
+
     @Test
     fun `can remove all global roles from a user`() {
       val userId = insertUser(10)
 
-      userStore.updateGlobalRoles(userId, emptySet())
+      userStore.updateGlobalRoles(setOf(userId), emptySet())
+
+      assertEquals(emptyList<Any>(), userGlobalRolesDao.findAll())
+    }
+
+    @Test
+    fun `can remove all global roles from a set of users`() {
+      val userId1 = insertUser(10)
+      val userId2 = insertUser(11)
+
+      userStore.updateGlobalRoles(setOf(userId1, userId2), emptySet())
```

Comment:
The previous test case has this problem too, but I'm just now noticing it: this doesn't actually test removing global roles because the test never inserts any global roles for `updateGlobalRoles` to remove.